scripts/test_mail-alarm.py

```diff
@@ -25,6 +26,24 @@ def mock_setup(module):
     module.branding.PRODUCT_BRAND = "XenServer"
     module.branding.BRAND_CONSOLE = "XenCenter"
 
+def get_alarm_xml(xmlalarm_str, xmlbody_str, xmlcls_str, xmlname_str):
```

Comment:
Yes. constant string will be better. But the function won't work like this.
Because the `XML_BODY_COMMON` is needed format ether.
I will refine this, thank you!